File: c/darts/darts.c
#include <math.h>
#include "darts.h"

uint8_t score(coordinate_t landing_position)
{
	float distance = hypotf(landing_position.x, landing_position.y);

	if (distance <= 1.0F)
		return 10;
	if (distance <= 5.0F)
		return 5;
	if (distance <= 10.0F)
		return 1;
	return 0;
}

File: python/darts/darts.py
from math import sqrt


def score(x: float, y: float) -> int:
    radius: float = sqrt(x ** 2 + y ** 2)

    if radius <= 1:
        return 10
    if radius <= 5:
        return 5
    if radius <= 10:
        return 1
    return 0
